pkg/chain/ethereum/ethereum.go

```diff
@@ -519,21 +517,19 @@ func (ec *EthereumChain) PastSignatureSubmittedEvents(
 	keepAddress string,
 	startBlock uint64,
 ) ([]*eth.SignatureSubmittedEvent, error) {
-	if !common.IsHexAddress(keepAddress) {
-		return nil, fmt.Errorf("invalid keep address: [%v]", keepAddress)
+	if !common.IsHexAddress(keepAddress) {	
+		return nil, fmt.Errorf("invalid keep address: [%v]", keepAddress)	
 	}
-
-	keepContractEventLog, err := eventlog.NewBondedECDSAKeepEventLog(
-		common.HexToAddress(keepAddress),
-		ec.client,
-	)
+	keepContract, err := ec.getKeepContract(common.HexToAddress(keepAddress))
 	if err != nil {
 		return nil, err
 	}
 
-	events, err := keepContractEventLog.PastSignatureSubmittedEvents(
+	// 2nd nil denotes latest block
+	events, err := keepContract.PastSignatureSubmittedEvents(
 		startBlock,
 		nil,
```

Comment:
Maybe we can put the comment about `// latest block` in this line?

Reply:
Sure.